trainer/vis.py

```diff
@@ -212,18 +218,32 @@ def store_raw_predictions(predictions: Dict[str, Any],
       panoptic_outputs[:, :, 2] = predicted_instance_labels % 256
       vis_utils.save_annotation(panoptic_outputs,
                                 output_folder,
-                                image_filename,
+                                panoptic_filename,
                                 add_colormap=False)
     elif raw_panoptic_format == 'two_channel_numpy_array':
       panoptic_outputs[:, :, 0] = predicted_semantic_labels
       panoptic_outputs[:, :, 1] = predicted_instance_labels
       with tf.io.gfile.GFile(
-          os.path.join(output_folder, image_filename + '.npy'), 'w') as f:
+          os.path.join(output_folder, panoptic_filename + '.npy'), 'w') as f:
         np.save(f, panoptic_outputs)
     else:
       raise ValueError(
           'Unknown raw_panoptic_format %s.' % raw_panoptic_format)
 
+  if common.PRED_DEPTH_KEY in predictions:
+    output_folder = os.path.join(save_dir, 'raw_depth')
+    if dataset_info.is_video_dataset:
+      output_folder = os.path.join(output_folder, sequence)
+      tf.io.gfile.makedirs(output_folder)
+    depth_outputs = predictions[common.PRED_DEPTH_KEY]
+    depth_outputs = np.squeeze(depth_outputs)
+    depth_outputs = depth_outputs.astype(np.float32) * 256
```

Comment:
Line 240 to 245

Could we please update vis_utils.save_annotation for this part?
Add new input argument scale_factor when scale_values is True (and defaults it to 255 for backward compatibility) and save_image_type (defaults to uint8) or something like that.

Reply:
Thanks for the suggestions! I feel like having a scale_factor would increase the code complexity. Maybe just leave it the current way?

Reply:
I see the point. Thanks for pointing it out.
How about we change `scale_values`?
We change it to `scale_value`, defaults to 1 which does not scale any value.

Reply:
The codes have been revised. PTAL. Thanks!